src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -259,6 +288,11 @@ export class MatCheckbox implements AfterViewInit, OnDestroy, ControlValueAccess
     // @breaking-change 10.0.0: Remove this after the `_clickAction` parameter is removed as an
     // injection parameter.
     this._clickAction = this._clickAction || this._options.clickAction;
+    console.log('potato!!!!!!!!!!!!!!!');
+  }
+
+  _getAttrBlocklist() {
```

Comment:
no need to bother with the getter method, just make `_attrBlockist` public

Reply:
If we make it public, I'd prefer we mark it as readonly and type it as a ReadonlySet